doc/using/Synthesis.rst

```diff
@@ -58,6 +58,37 @@ See :ghdlsharp:`1174` for on-going discussion about other output formats.
 .. TIP::
    Files can be provided in any order.
 
+.. _synthesis_options:
+
+Synthesis options
+==================
+
+In addition to analyze options, you can use following synthesis specific options:
+
+.. option:: -gNAME=VALUE
```

Comment:
This is a run option too: https://ghdl.github.io/ghdl/using/Simulation.html#cmdoption-ghdl-gGENERIC

On the one hand, I don't know if the prototype should be the same ( `-gGENERIC=VALUE` vs `-gNAME=VALUE`). It might produce a collision.

On the other hand, I don't know if the comments/notes about this option changing to an analysis option in the future applies to synthesis too.

In any case, I believe a link/ref should exist from this section to the other one. The types and syntax which are supported are likely the same in both cases.

Reply:
I added a link to the run option.

Reply:
Maybe you could check the link, as it isn't generating a link but only text?

Reply:
I fixed it: https://github.com/ghdl/ghdl/commit/1b23af06d9806867887aebad36b71c26e10584c1. The key needs to be `-gGENERIC`.